tests/test_avro_schema.py

```diff
@@ -8,6 +8,206 @@
 import json
 import pytest
 
+# Schemas defined in AvroCompatibilityTest.java. Used here to ensure compatibility with the schema-registry
+schema1 = parse('{"type":"record","name":"myrecord","fields":[{"type":"string","name":"f1"}]}')
+schema2 = parse(
+    '{"type":"record","name":"myrecord","fields":[{"type":"string","name":"f1"},{"type":"string",'
+    '"name":"f2","default":"foo"}]}'
+)
+schema3 = parse('{"type":"record","name":"myrecord","fields":[{"type":"string","name":"f1"},{"type":"string","name":"f2"}]}')
+schema4 = parse('{"type":"record","name":"myrecord","fields":[{"type":"string","name":"f1_new","aliases":["f1"]}]}')
```

Comment:
Any failures related to this could make sense, as we only compare field names at the moment, instead of aliases